integration/Rakefile

```diff
@@ -1,13 +1,38 @@
 require 'rototiller'
 
+namespace :ci do
+  namespace :test do
+
+    desc 'Tests at the component level for the pe-r10k project'
+    task :component => [:check_client_tool_env_vars] do
```

Comment:
Any verdict on that? And also, not sure where :check_client_tool_env_vars comes from? Will this be passed in to the test call in CI? And should I rename this to be r10k specific?